src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -787,6 +795,15 @@ internal async Task InvokeResponse(InvocationResponse invokeResponse)
             if (_executingInvocations.TryRemove(invokeResponse.InvocationId, out ScriptInvocationContext context)
                 && invokeResponse.Result.IsInvocationSuccess(context.ResultSource, capabilityEnabled))
             {
+                var currentInvocationLatency = _workerInvocationStopWatch.GetElapsedTime();
+                _workerChannelLogger.LogDebug($"[HostMonitor] Worker {Id} invocation request took {currentInvocationLatency.TotalMilliseconds}ms");
+
+                int successfulInvocations = WorkerInvocationMetrics.IncrementSuccessfulInvocationsOfWorker(Id, _invocationMetricsPerWorkerId);
+                double averageInvocationLatency = WorkerInvocationMetrics.UpdateAverageInvocationLatency(Id, _invocationMetricsPerWorkerId, currentInvocationLatency);
```

Comment:
How would this data (invocation latency) differ from the `function.invoke.latency` metric/event which we already capture and store in FunctionsMetrics?

Reply:
Removing latency logic for now as it was not the performance efficient approach and anyway was not using it for faulty worker calculation. Will investigate the correct approach there and then update.